ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -346,7 +359,7 @@ public void ExportToFile(string resourceName, string fileName, VModel model)
                 if (string.IsNullOrEmpty(meshReference))
                 {
                     // If refmesh is null, take an embedded mesh
-                    meshes[i] = (embeddedMeshes[embeddedMeshIndex++], $"{modelName}.Embedded.{embeddedMeshIndex}");
```

Comment:
Just checking: Does this not cause name collisions in files with multiple models with embedded meshes?

Reply:
As far as I can tell, glTF doesn't really care about name collisions.

Either way, I removed `modelName` from embedded meshes names due to models now having their own nodes - this way there are less `WorldMatrix`es in resulting glTF, and it's less bloated when there are multiple models present.
https://github.com/SteamDatabase/ValveResourceFormat/pull/467/files#diff-a740f76f43bd153e9126ae3abe86209608d28825f381ed37b2d74aa3bcbc9c93R278
![image](https://user-images.githubusercontent.com/11695747/206853854-3559016d-0170-41e5-aecf-3a737771e63f.png)